package.json

```diff
@@ -114,8 +114,8 @@
     "cozy-scanner": "^0.3.2",
     "cozy-scripts": "1.13.2",
     "cozy-sharing": "1.9.1",
-    "cozy-stack-client": "13.4.2",
-    "cozy-ui": "35.7.3",
+    "cozy-stack-client": "13.5.1",
+    "cozy-ui": "35.8.0",
```

Comment:
it's 35.9.0 right? https://github.com/cozy/cozy-ui/releases/tag/v35.9.0

Reply:
Merci !